CHANGES/2303.misc

```diff
@@ -0,0 +1 @@
+Added documentation steps to remove content.
```

Comment:
I would make it a `.doc`

Reply:
Done!
In this case, I think that would be a good idea to also update our [pulpcore docs](https://docs.pulpproject.org/pulpcore/contributing/git.html#changelog-update) (there is no mention to .doc extension), right?